lib/extensions/string_replace_dot_remote_config.dart

```diff
@@ -0,0 +1,6 @@
+extension ReplaceDotInRemoteConfigString on String {
+  String replaceDotInRemoteConfig() {
+    String stringWithoutDot = replaceAll('remote_config_', 'remote_config.');
+    return stringWithoutDot;
+  }
+}
```

Comment:
Make a generic replaceUnderscoreToDot methods for possible future uses. ( use the 'this' to get the current string value)